src/invert/laplace/invert_laplace.cxx

```diff
@@ -71,45 +71,56 @@ Laplacian::Laplacian(Options *options, const CELL_LOC loc, Mesh *mesh_in)
   // Communication option. Controls if asyncronous sends are used
   async_send = (*options)["async"].doc("Use asyncronous MPI send?").withDefault(true);
 
-  BoutReal filter = (*options)["filter"]
-                        .doc("Fraction of Z modes to filter out. Between 0 and 1")
-                        .withDefault(0.0);
-  int ncz = localmesh->LocalNz;
+  const BoutReal filter = (*options)["filter"]
+                              .doc("Fraction of Z modes to filter out. Between 0 and 1")
+                              .withDefault(0.0);
+  const int ncz = localmesh->LocalNz;
   // convert filtering into an integer number of modes
-  maxmode = ROUND((1.0 - filter) * static_cast<BoutReal>(ncz / 2));
-  // Can be overriden by max_mode option
-  OPTION(options, maxmode, maxmode);
-  if(maxmode < 0) maxmode = 0;
-  if(maxmode > ncz/2) maxmode = ncz/2;
+  maxmode = (*options)["maxmode"]
+                .doc("The maximum Z mode to solve for")
+                .withDefault(ROUND((1.0 - filter) * static_cast<BoutReal>(ncz / 2)));
 
-  OPTION(options, low_mem, false);
+  // Clamp maxmode between 0 and nz/2
+  if (maxmode < 0) {
+    maxmode = 0;
+  }
+  if (maxmode > ncz / 2) {
+    maxmode = ncz / 2;
+  }
+
+  low_mem = (*options)["low_mem"]
+                .doc("If true, reduce the amount of memory used")
+                .withDefault(false);
```

Comment:
It looks like `low_mem` is only actually implemented now by `LaplacePDD`. If we remove that (#1828) we can remove the option too.